src/search.c

```diff
@@ -519,8 +525,15 @@ int search(Thread *thread, PVariation *pv, int alpha, int beta, int depth) {
 
         // Step 16B (~2 elo). Noisy Late Move Reductions. The same as Step 15A, but
         // only applied to Tactical moves with unusually poor Capture History scores
-        else if (!isQuiet && depth > 2 && played > 1)
-            R = MIN(depth - 1, MAX(1, MIN(3, 3 - (hist + 4000) / 2000)));
+        else if (!isQuiet && depth > 2 && played > 1) {
+            R = MIN(3, 3 - (hist + 4000) / 2000);
```

Comment:
Lets add a new line after the else if line. Also, now is a good time to comment this a bit. Something like ``// Set R based on Capture History``. Might be fair to change that elo estimate of ~2 to ~3, if you desire.